.solargraph.yml

```diff
@@ -0,0 +1,2 @@
+reporters:
+  - rubocop
```

Comment:
what is this solargraph for? is new to me

Reply:
I plan on getting linting on the steps too, solargraph can be used with most IDE